src/codewhisperer/tracker/codewhispererCodeCoverageTracker.ts

```diff
@@ -112,17 +106,25 @@ export class CodeWhispererCodeCoverageTracker {
         if (vscode.window.activeTextEditor) {
             this.updateAcceptedTokensCount(vscode.window.activeTextEditor)
         }
+        // the accepted characters without counting user modification
         let acceptedTokens = 0
+        // the accepted characters after calculating user modificaiton
+        let unmodifiedAcceptedTokens = 0
         for (const filename in this._acceptedTokens) {
             this._acceptedTokens[filename].forEach(v => {
                 if (filename in this._totalTokens && this._totalTokens[filename] >= v.accepted) {
-                    acceptedTokens += v.accepted
+                    unmodifiedAcceptedTokens += v.accepted
+                    acceptedTokens += v.text.length
                 }
             })
         }
         const percentCount = ((acceptedTokens / totalTokens) * 100).toFixed(2)
         const percentage = Math.round(parseInt(percentCount))
         const selectedCustomization = getSelectedCustomization()
+        if (this._serviceInvocationCount <= 0) {
+            getLogger().info(`Skip emiting code contribution metric`)
```

Comment:
should this be debug level, or is it useful for customers?

Reply:
changed to debug